RetailCoder.VBE/Inspections/UntypedFunctionUsageInspectionResult.cs

```diff
@@ -32,18 +32,20 @@ public override string Description
     public class UntypedFunctionUsageQuickFix : CodeInspectionQuickFix
     {
         public UntypedFunctionUsageQuickFix(ParserRuleContext context, QualifiedSelection selection) 
-            : base(context, selection, string.Format(InspectionsUI.QuickFixUseTypedFunction_, context.GetText(), context.GetText() + "$"))
+            : base(context, selection,
+                  string.Format(InspectionsUI.QuickFixUseTypedFunction_,
+                      context.GetText(), context.GetText().Insert(context.GetText().IndexOf('('), "$")))
```

Comment:
This seems to be blowing up with #1537 now ... Might need to check our assumptions here